e2e/wdio/platform/tests/date-time-picker.e2e-spec.ts

```diff
@@ -237,17 +238,49 @@ describe('Datetime picker suite', () => {
         click(okButton);
         expect(getValue(datePickerInput)).toEqual(date);
     });
-});
 
-it('should check that OK buttons have correct text', () => {
-    const datepickerButtonsLength = getElementArrayLength(datePickerButton);
-    for (let i = 0; i < datepickerButtonsLength; i++) {
-        if (!getElementClass(datePickerButton, i).includes('disabled')) {
-            click(datePickerButton, i);
-            expect(getText(okButton + buttonText)).toEqual('OK');
-            click(okButton);
+    it('should check that OK buttons have correct text', () => {
+        const datepickerButtonsLength = getElementArrayLength(datePickerButton);
+        for (let i = 0; i < datepickerButtonsLength; i++) {
+            if (!getElementClass(datePickerButton, i).includes('disabled')) {
+                click(datePickerButton, i);
+                expect(getText(okButton + buttonText)).toEqual('OK');
+                click(okButton);
+            }
         }
-    }
+    });
+
+    // skipped due to https://github.com/SAP/fundamental-ngx/issues/7112
+    xit('should check that date-time picker does not have error if it contains valid value', () => {
+        scrollIntoView(inputGroup, 8);
+        let validDate;
+        const currentDate = new Date();
+        const currentMonth = currentDate.getMonth() + 1;
+        const currentDay = currentDate.getDate();
+        const currentYear = currentDate.getFullYear();
+        let currentMinute = currentDate.getMinutes().toString();
+        if (currentMinute === '0') {
+            currentMinute = '00';
+        } else if (parseInt(currentMinute) < 10) {
+            currentMinute = '0' + currentMinute;
+        }
+        const currentHour = currentDate.toLocaleString('en-US', { timeZone: 'UTC', hour: 'numeric', hour12: true });
+
+        if (currentHour[1] === ' ') {
+            validDate = `${currentMonth}/${currentDay}/${currentYear}, ${
+                currentHour[0]
+            }:${currentMinute} ${currentHour.slice(2, 4)}`;
+        }
+        if (currentHour[1] !== ' ') {
```

Comment:
You can use just `else`

Reply:
Had a conversation with @N1XUS and @Seamoo13 and decided to keep it as it is